docs/getting-started/intro.md

```diff
@@ -8,6 +8,9 @@ Welcome to Tauri!
 
 Tauri is a polyglot and generic system that is very composable and allows engineers to make a wide variety of applications. It is used for building applications for Desktop Computers using a combination of [Rust](https://www.rust-lang.org/) tools and HTML rendered in a Webview. Apps built with Tauri can ship with any number of pieces of an optional JS API / Rust API so that webviews can control the system via message passing.
 
+Anything that can be displayed on a website, can be displayed in a native Tauri app!
```

Comment:
```suggestion
Anything that can be displayed on a website, can be displayed in a OS-native Tauri app!
```
Perhaps further saying OS? to completely let the understand the tie in in first paragraph with "Desktop Computers"?

Reply:
We don't use "native" word; many people made us understand that since we're using web technologies, it wasn't native. 
I tend to use "desktop"  instead (until we get mobile, then I'd use "desktop/mobile").

Reply:
OK